.github/workflows/release.yml

```diff
@@ -13,7 +13,7 @@ on:
 name: Release
 
 env:
-  GO_VERSION: "1.23.5"
```

Comment:
I think we should use 1.23.6 instead of latest version for support release.

Reply:
Yes. :100:  I think this was a miss when we made 2.0 branch cut. In other release branches we stick to the older stable of golang for making the release.